client/src/com/mirth/connect/client/ui/editors/BaseEditorPane.java

```diff
@@ -85,6 +85,12 @@
 import javax.swing.tree.TreePath;
 import javax.swing.tree.TreeSelectionModel;
 
+import com.mirth.connect.client.ui.BrandingConstants;
+import com.mirth.connect.client.ui.Frame;
+import com.mirth.connect.client.ui.Mirth;
+import com.mirth.connect.client.ui.PlatformUI;
+import com.mirth.connect.client.ui.SortableTreeTableModel;
+import com.mirth.connect.client.ui.UIConstants;
```

Comment:
chore: remove unnecessary change

I know you needed to add BrandingConstants, but I'm not sure what caused everything else to move from the original location. The imports were already sorted and are now out of place.